docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+
+  postgre:
```

Comment:
```suggestion
  postgres:
```

Reply:
Please, read my reply in the previous comment

Reply:
[Wikipedia](https://en.wikipedia.org/wiki/PostgreSQL) tells me that `Postgres` is also correct:

```
PostgreSQL also known as Postgres...
```

I will change it :)

Reply:
Got it ! Kindly ignore this.